test/test.js

```diff
@@ -14,6 +14,8 @@
  * limitations under the License.
  */
 
+'use strict';
```

Comment:
why do we have it here? do we need to use 'use strict' in all of the source code?

Reply:
@vsemozhetbyt yeah, let's exclude

Reply:
Done.